modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/manager/FeatureFlagPreferencesManager.java

```diff
@@ -31,6 +32,13 @@
 public class FeatureFlagPreferencesManager {
 
 	public Boolean isEnabled(long companyId, String key) {
+		if (Validator.isNull(
```

Comment:
@yuhai I do not understand why this is related to this change "The issue is exposed after LPS-176550(https://github.com/liferay-core-infra/liferay-portal/commit/b289f4a087f7456de1220f6d7666463f80ea222b). When delete an instance from UI, PortalPreferences data can't be deleted."